lib/Sema/DerivedConformanceDifferentiable.cpp

```diff
@@ -646,13 +646,11 @@ getOrSynthesizeTangentVectorStruct(DerivedConformance &derived, Identifier id) {
           C, /*implicit*/ true, SourceLoc(), SourceLoc(),
           /*linear*/ false, {}, None, None, derivativeGenSig);
       member->getAttrs().add(diffableAttr);
-      // Compute getter parameter indices.
-      auto *getterType = member->getAccessor(AccessorKind::Get)
-                             ->getInterfaceType()
-                             ->castTo<AnyFunctionType>();
-      AutoDiffIndexSubsetBuilder builder(getterType);
-      builder.setParameter(0);
-      diffableAttr->setParameterIndices(builder.build(C));
+      // Set getter `@differentiable` attribute parameter indices.
+      llvm::SmallBitVector parameters(1);
```

Comment:
If you do not need to set bits dynamically in a loop, call `AutoDiffIndexSubset::get` directly to create a subset.
```cpp
AutoDiffIndexSubset::get(C, 1, {0})
```

Reply:
Done in https://github.com/apple/swift/pull/27514/commits/ea737dd2453c59d94f157b8f117f7986ca29c94e, thanks.